traitsui/testing/tester/wx/helpers.py

```diff
@@ -29,3 +91,73 @@ def mouse_click_button(control, delay):
         wx.wxEVT_COMMAND_BUTTON_CLICKED, control.GetId()
     )
     control.ProcessEvent(click_event)
+
+def mouse_click_text_ctrl(control, delay):
+    """ Performs a mouce click on a wxTextCtrl.
+
+    Parameters
+    ----------
+    control : wxObject
+        The wx Object to be clicked.
+    delay: int
+        Time delay (in ms) in which click will be performed.
+    """
+    if not control.IsEnabled():
+        return
+    if not control.HasFocus():
+        control.SetFocus()
+    wx.MilliSleep(delay)
+    click_event = wx.CommandEvent(
+        wx.wxEVT_COMMAND_LEFT_CLICK, control.GetId()
+    )
+    control.ProcessEvent(click_event)
+
+
+def key_click_text_ctrl(control, interaction, delay):
+    """ Performs simulated typing of a key on the given wxObject
+    after a delay.
+
+    Parameters
+    ----------
+    control : wxObject
+        The wx Object to be acted on. Should be wx.TextCtrl instance. If it is
+        a wx.StaticText (i.e. from a ReadonlyEditor) nothing occurs.
+    interaction : instance of command.KeyClick
+        The interaction object holding the key input
+        to be simulated being typed
+    delay : int
+        Time delay (in ms) in which the key click will be performed.
+    """
+    if isinstance(control, wx.StaticText):
+        return
```

Comment:
See another comment: It is questionable to support key click and key sequence for StaticText.  I think we can just remove this for now. Same for key_sequence_text_ctrl.